app/buck2_common/src/http/retries.rs

```diff
@@ -64,11 +75,18 @@ where
         if let Some(http_error) = http_error {
             if http_error.is_retryable() {
                 if let Some(b) = backoff.peek() {
-                    tracing::warn!(
-                        "Retrying a HTTP error after {} seconds: {:#}",
-                        b.as_secs(),
-                        http_error
-                    );
+                    let secs = b.as_secs();
+                    if secs > 30 {
+                        // NOTE (aseipp): 30 seconds is arbitrary for this, but
+                        // at that point things will be pretty annoying!
+                        tracing::warn!(
+                            host = host,
+                            retries = retries,
+                            "Egregious HTTP backoff - retrying slow request in {} seconds",
+                            secs,
```

Comment:
- Can we avoid calling the backoff "egregious"? 
- Also, do note we are not retrying a slow request here we're rather retrying a failed one
- We shouldn't hide the error here